Projects/Team1/Semeniuk/Feature_with_project_class_Animal_Fish/StarProject/Collection.cs

```diff
@@ -106,9 +111,10 @@ public static void ReadXML(List<Animals> AnimalCollection)
             }
         }
         //функція консольного виведення полів і значень тварин старших за n
-        static void PrintOlderThen(List<Animals> AnimalCollection, int YearsOld)
+        public static string PrintOlderThen(List<Animals> Collection, int YearsOld)
```

Comment:
not Collection and YearsOld but collection and yearsOld
we need to use lowercase for parameters names

Reply:
Got it